Makefile

```diff
@@ -99,7 +99,7 @@ kind-cluster: ## Standup a kind cluster for e2e testing usage
 install-apis: generate kustomize ## Install the core rukpak CRDs
 	$(KUSTOMIZE) build manifests/apis/crds | kubectl apply -f -
 
-install-provisioners: kustomize ## Install the plain provisioner
+install-provisioners: kustomize ## Install the rukpak provisioners
```

Comment:
Seems unrelated, but I'll allow it. :)

Reply:
I'm usually pretty good about commits but I thought this was a solid change to sneak that in after I missed it in #260 :upside_down_face: